content/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/aggregatewindow.md

```diff
@@ -11,8 +11,8 @@ menu:
 weight: 501
 related:
   - /v2.0/query-data/flux/window-aggregate/
-  - https://docs.influxdata.com/influxdb/latest/query_language/functions/#aggregations – InfluxQL – Aggregate functions
-  - https://docs.influxdata.com/influxdb/latest/query_language/data_exploration/#the-group-by-clause – InfluxQL – GROUP BY time()
+  - https://docs.influxdata.com/influxdb/latest/query_language/functions/#aggregations, InfluxQL – Aggregate functions
+  - https://docs.influxdata.com/influxdb/latest/query_language/data_exploration/#the-group-by-clause, InfluxQL – GROUP BY time()
```

Comment:
Second link isn't going directly to #the-group-by-clause...looks like we need to fix "The GROUP BY clause" in v1 (currently an H1)

Reply:
This link was redirected because the file name was changed (`data_exlporation` => `explore_data`) and the redirect rule removed the anchor. I updated the link so it points to the new URL and the anchor works.